src/mozanalysis/experiment.py

```diff
@@ -369,30 +369,19 @@ def build_enrollments_query(
         Returns:
             A string containing a BigQuery SQL expression.
         """
-        analysis_windows_query = self._build_analysis_windows_query(
-            time_limits.analysis_windows
-        )
-
         enrollments_query = custom_enrollments_query or self._build_enrollments_query(
             time_limits, enrollments_query_type
         )
 
         segments_query = self._build_segments_query(segment_list, time_limits)
 
         return """
-            WITH analysis_windows AS (
-                {analysis_windows_query}
-            ),
-            raw_enrollments AS ({enrollments_query}),
+            WITH raw_enrollments AS ({enrollments_query}),
             segmented_enrollments AS ({segments_query})
 
-            SELECT
-                se.*,
-                aw.*
```

Comment:
The `analysis_window_start` and `analysis_window_end` fields will not be part of the enrollments tables anymore? Are there any (query) dependencies that I'm missing that would break once these are gone?